app/views/jobs/show.html.slim

```diff
@@ -1,16 +1,24 @@
-#job.job-posting
-  .job-posting__header
-    h2.job-company=@job.company
-    h1.job-title=@job.title
-    span.job-posting-date=@job.posted_at
-    span.job-availability=@job.availability
-    .job-share=mail_to "", "Share this job", 
-      {subject: "[Chadev+Jobs] #{@job.title}", 
-        body: "Here is a job I thought you might be interested in. <br/><br/> #{job_url(@job)}"}
+- if @job.akismet_spam
```

Comment:
I suppose we could wrap it with a predicate method. Alternatively, I thought about this earlier today and I was thinking it might be possible to simply rename this attribute to `spam`, at which point the included `Rakismet` model would pick up on it (hopefully) https://github.com/joshfrench/rakismet/blob/master/lib/rakismet/model.rb#L37

...then we could ask for the value by calling `.spam?`